### ISSUE TITLE
Lastets spectrum scanner bugs

### ISSUE THREAD

**Issue by rizomorfo** [piotr022/UV_K5_playground#10]
If you open the scanner and close it pressing TX works, fine. But it you do it 2 times it always goes back to the scanner screen. I have to turn the radio off to fix it.
Also if you start the scanner while RX the speakers go BEEEEEEEEEEEEP

Anyway is veeery coool addon. 
Congrats!

**Comment by piotr022** [comment]
hi, check new release, fixed exiting from view, and added zoom in/out, resolution up/down features :D

**Comment by rizomorfo** [comment]
WOWWW muuuch better. You are a genius man! hat off.
I think this is the most usefull mod!!! The Smeter is great too. Wish we can have both! 
Congrats man!

**Comment by LZ2DMV** [comment]
Yes, would be great if you can ship a release with spectrum + RSSI SBar + Max AM and FM frequencies unlocked.

**Comment by Kemon2023** [comment]
it is better now.... very much thanks @piotr022

**Comment by lohtse** [comment]
> Yes, would be great if you can ship a release with spectrum + RSSI SBar + Max AM and FM frequencies unlocked.

That would be a damn good one

**Comment by JEDI-HACKER** [comment]
For some reason I can't get the sbar to work after flashing a few times.

**Comment by whytf** [comment]
> Yes, would be great if you can ship a release with spectrum + RSSI SBar + Max AM and FM frequencies unlocked.

would appreciate that release as well
